papi: fix ci

haampie opened the pull request:
`papi@7.1%cce` is currently broken in Gitlab CI cause the `-ffree-form` flag isn't recognized.

haampie committed:
papi: fix ci

alecbcs commented on var/spack/repos/builtin/packages/papi/package.py:
```diff
@@ -72,6 +72,7 @@ class Papi(AutotoolsPackage, ROCmPackage):
     conflicts("%gcc@8:", when="@5.3.0", msg="Requires GCC version less than 8.0")
     conflicts("+sde", when="@:5", msg="Software defined events (SDE) added in 6.0.0")
     conflicts("^cuda", when="@:5", msg="CUDA support for versions < 6.0.0 not implemented")
+    conflicts("%cce", when="@7.1", msg="-ffree-form flag not recognized")
```
```suggestion
    conflicts("%cce", when="@7.1:", msg="-ffree-form flag not recognized")
```
Unless we think this'll be removed/fixed the the next version, I might suggest conflicting with `%cce` until it is resolved (unlimited upper bound) so version bumps don't break CI.

bernhardkaindl committed:
Apply suggestion to conflic with %cce@7.1: until it is resolved.

By @alecbcs:

Unless we think this'll be removed/fixed the the next version, I might suggest conflicting with %cce until it is resolved (unlimited upper bound) so version bumps don't break CI.

Agreed!

Co-authored-by: Alec Scott <alec@bcs.sh>

bernhardkaindl reviewed (approved):
alalazo referred to @alecbcs, who suggested:

Unless we think this'll be removed/fixed the the next version, I might suggest conflicting with %cce until it is resolved (unlimited upper bound) so version bumps don't break CI.

Agreed!

I applied to GitHub-PR-suggestion by @alecbcs and approve accordingly.

Will try to enable auto-merge as well to auto-merge when GitLab CI passes.
